fetch-license.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+curl -o LICENSE-rtsp-simple-server https://raw.githubusercontent.com/aler9/rtsp-simple-server/main/LICENSE
```

Comment:
I putting the curl command (without the `-o`) in the Dockerfile itself, in rtsp stage, and renaming it in the last stage when copying to the final stage.

It's a bit of an overkill to have a script with this command.

Reply:
If you insist on keeping it in a script, then the script should be under `bin` and have a more specific name, e.g. `bin/fetch-rtsp-license.sh`.

Reply:
Thanks @farshidtz
Removed the script. However, it seems that we can only get the license file at the builder stage. Because rtsp-simple-server image is based on `scrach`: https://github.com/aler9/rtsp-simple-server/blob/f7c08f577af3db5032393b9e840dc43c0b385e0c/Makefile#L230

Reply:
Thanks. That's okay then. 
In the meantime, I'll check if I can upstream the license addition.

Reply:
Thanks. That's okay then. 
In the meantime, I'll check if I can upstream the license addition.